package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mattermost-mobile",
-  "version": "1.27.0",
+  "version": "1.27.1",
```

Comment:
Glad that this diff is making its way to master via this commit too. I'd noticed it come up on current `master`.
How come it didn't make it as part of the version bump commit? Did I miss something in review?